crates/node/service/src/actors/l1_watcher_rpc.rs

```diff
@@ -94,7 +120,30 @@ impl NodeActor for L1WatcherRpc {
                     Some(new_head) => {
                         // Send the head update event to all consumers.
                         let head_block_info = self.block_info_by_hash(new_head).await?;
-                        self.sender.send(head_block_info)?;
+                        self.head_sender.send(head_block_info)?;
+
+                        // For each log, attempt to construct a `SystemConfigLog`.
+                        // Build the `SystemConfigUpdate` from the log.
+                        // If the update is an Unsafe block signer update, send the address
+                        // to the block signer sender.
+                        let logs = self.fetch_logs(new_head).await?;
+                        let ecotone_active = self.config.is_ecotone_active(head_block_info.timestamp);
+                        for log in logs {
+                            let sys_cfg_log = SystemConfigLog::new(log.into(), ecotone_active);
+                            if let Ok(SystemConfigUpdate::UnsafeBlockSigner(UnsafeBlockSignerUpdate { unsafe_block_signer })) = sys_cfg_log.build() {
+                                info!(
+                                    target: "l1_watcher",
+                                    "Unsafe block signer update: {unsafe_block_signer}"
+                                );
+                                if let Err(e) = self.block_signer_sender.send(unsafe_block_signer) {
+                                    error!(
+                                        target: "l1_watcher",
+                                        "Error sending unsafe block signer update: {e}"
+                                    );
+                                }
+                            }
+                        }
+
```

Comment:
```suggestion
```

Reply:
The event might be registered by the derivation pipeline, but the log update was never actually parsed from the `Log` instance.